src/sentry/static/sentry/app/views/releases/detail/releaseHeader.tsx

```diff
@@ -114,11 +122,34 @@ const ReleaseHeader = ({
               <Count value={sessionsCrashed} />
             </ReleaseStat>
           )}
+          <Feature features={['release-performance-views']}>
+            <ReleaseStat
+              label={t('Apdex Score')}
```

Comment:
I agree with Ben on this. We don't use 'Score' anywhere else when referring to Apdex.

Reply:
I don't mind changing this at all to just `Apdex`, but we do use `score` in the transaction summary page:
https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/views/performance/transactionSummary/userStats.tsx#L85